resources/uk/gov/hmcts/renovate/renovate-config-check.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -x
+
+if [[ ! -f .github/renovate.json ]]
```

Comment:
i wouldn't enforce this location, renovate can be in a number of places, I would at least support .github/renovate.json and renovate.json.

If renovate automatically onboarded a team by sending them a PR it does it to renovate.json